lib/Persistence/TrafficLimiter.php

```diff
@@ -83,6 +107,39 @@ public static function getHash($algo = 'sha512')
         return hash_hmac($algo, $_SERVER[self::$_ipKey], ServerSalt::get());
     }
 
+    /**
+     * Validate $_ipKey against configured ipranges. If matched ratelimiter will ignore ip
+     *
+     * @access private
+     * @static
+     * @param  string $ipRange
+     * @return bool
+     */
+    private static function matchIp($ipRange = null)
+    {
+        // Match $_ipKey to $ipRange and if it matches it will return with a true
+        $address = \IPLib\Factory::addressFromString($_SERVER[self::$_ipKey]);
+        $range   = \IPLib\Factory::rangeFromString(trim($ipRange));
+        // If $range is null something went wrong (possible invalid ip given in config)
+        if ($range == null) {
+            return false;
+        } else {
+            // Ip-lib does throws and exception when something goes wrong, if so we want to catch it and set contained to false
+            try {
+                $contained = $address->matches($range);
```

Comment:
I think you can probably just shorten this line to:
```php
                 return $address->matches($range);
```
If an exception occurs, it prevents the return and gets caught. It would obsolete lines 135-140.